src/index.spec.ts

```diff
@@ -20,6 +20,17 @@ describe('ts-node', function () {
   })
 
   describe('cli', function () {
+    it('should not react to signals, but forward them to the child process', function (done) {
+      const proc = exec(`${BIN_EXEC} tests/signals`, function (err, stdout) {
+        expect(err).to.equal(null)
+        expect(stdout).to.equal('exit')
+
+        return done()
+      })
+
+      setTimeout(() => proc.kill('SIGTERM'), 100)
```

Comment:
For some reasons, the kill signal is not properly sent.

Reply:
Would it work if you forwarded the signals to the child? It's possible that the current code change is just ignoring the signal.